src/ui/Tab/Tab.ts

```diff
@@ -116,6 +118,11 @@ export class Tab extends Component {
     this.bind.onQueryState(ModelEvents.CHANGE_ONE, QueryStateAttributes.T, (args: IAttributeChangedEventArg) => this.handleQueryStateChanged(args));
     this.bind.on(element, 'click', (e: Event) => this.handleClick());
     this.render();
+      
+      if(Tab.isFirstTab) {
+        Tab.isFirstTab = false;
+        ResponsiveTabs.init();

```

Comment:
The init call is probably where you will need to pass the reference that ActiveTab needs to do it's job : the root of the current interface.